routes/routes.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
don't use 
```
use strict
```

it's and ECMAScript 5 implementation to prevent some roles like shadowing (reassign value to variable)
instead you can use ESlint to force some roles on the development team.